llvm/utils/UpdateTestChecks/common.py

```diff
@@ -74,13 +74,17 @@ class Filter(Regex):
 
     """
 
-    def __init__(self, regex, is_filter_out):
+    def __init__(self, regex, is_filter_out, is_filter_all_after):
```

Comment:
It's probably best to create a new, separate review for the change to the script, with a dedicated test for the script.

Reply:
Yep, can do. I just thought I'd put the whole thing here first for context and to help reviewers understand the typical use case. I'll spin out the python changes in a new PR.

Reply:
I've put up https://github.com/llvm/llvm-project/pull/129739